tools/Az.Tools.Predictor/Az.Tools.Predictor/TelemetryData.cs

```diff
@@ -0,0 +1,179 @@
+// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Management.Automation.Language;
+
+namespace Microsoft.Azure.PowerShell.Tools.AzPredictor
```

Comment:
Is it normal to have many different classes in the same file?

Reply:
I remove `TelemetryData` and move the other interface/class to seperate files.